5_People/career-technical.md

```diff
@@ -6,40 +6,41 @@ The top two positions are two branches of the technical career:
 * Lead Software Engineer careers means you start taking over managerial tasks and leading people.
 * Principal Software Engineer means you take ownership of all of technical nitty-gritty details in the company. 
 
-## Junior Programmer
+## Junior Programmer / Sysadmin
 
 * Work is thoroughly reviewed with substantial back and forth frequently needed before merging.
-* Basic language features are mastered, but some advanced structures may still be unfamiliar.
+* Basic language/technology features are mastered, but some advanced approaches may still be unfamiliar.
 * Occasional issues following patterns and approaches within existing code bases.
 * Works mostly on clearly defined and scoped individual features or problems.
-* Usually less than 2 years of programming experience.
+* Usually less than 2 years of technical experience.
 
 
-## Junior Software Engineer
+## Junior Software Engineer / DevOps
 
 * Work doesn't necessarily need to be reviewed, but general approach may be.
-* Follows established patterns and approaches within existing code bases with ease.
-* Is able to work on problems and features that are not clearly defined. Comes up with some specifications and API designs on their own.
-* Is able to take problem all the way from research, writing a User Story, implementation, and deployment.
-* Usually at least 2-5 years of programming experience.
+* Follows established patterns and approaches within established best practices bases with ease.
```

Comment:
This one sounds wrong now. Maybe the previous version but without `within`?
Like: `Follows established patterns and approaches existing code bases with ease.`
I don't think that one is too specific and I think it also applies to DevOps.